src/category_theory/limits/preserves/shapes/biproducts.lean

```diff
@@ -190,7 +233,55 @@ rfl
 end bicone
 
 variables (F : C ⥤ D) [preserves_zero_morphisms F] (X Y : C) [has_binary_biproduct X Y]
-  [preserves_binary_biproduct X Y F]
+
+section
+variables [has_binary_biproduct (F.obj X) (F.obj Y)]
+
+/-- As for products, any functor between categories with binary biproducts gives rise to a
+    morphism `F.obj (X ⊞ Y) ⟶ F.obj X ⊞ F.obj Y`. -/
+def biprod_comparison : F.obj (X ⊞ Y) ⟶ F.obj X ⊞ F.obj Y :=
+biprod.lift (F.map biprod.fst) (F.map biprod.snd)
+
+@[simp, reassoc] lemma biprod_comparison_fst :
+  biprod_comparison F X Y ≫ biprod.fst = F.map biprod.fst :=
+biprod.lift_fst _ _
+
+@[simp, reassoc] lemma biprod_comparison_snd :
+  biprod_comparison F X Y ≫ biprod.snd = F.map biprod.snd :=
+biprod.lift_snd _ _
+
+/-- As for coproducts, any functor between categories with binary biproducts gives rise to a
+    morphism `F.obj X ⊞ F.obj Y ⟶ F.obj (X ⊞ Y)`. -/
+def biprod_comparison' : F.obj X ⊞ F.obj Y ⟶ F.obj (X ⊞ Y) :=
+biprod.desc (F.map biprod.inl) (F.map biprod.inr)
+
+@[simp, reassoc] lemma inl_biprod_comparison' :
+  biprod.inl ≫ biprod_comparison' F X Y = F.map biprod.inl :=
+biprod.inl_desc _ _
+
+@[simp, reassoc] lemma inr_biprod_comparison' :
+  biprod.inr ≫ biprod_comparison' F X Y = F.map biprod.inr :=
+biprod.inr_desc _ _
+
+@[simp, reassoc] lemma biprod_comparison'_comp_biprod_comparison :
+  biprod_comparison' F X Y ≫ biprod_comparison F X Y = 𝟙 (F.obj X ⊞ F.obj Y) :=
```

Comment:
I presume this is an easy question for someone who knows more about biproducts than I do, but is the composition in the opposite direction false in general? Can we have a comment to that effect somewhere?

Reply:
By the results of this PR, the other composition is equal to the identity if and only if `F` preserves the biproduct. I have added a comment explaining this.